core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -67,6 +67,9 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
   if (loadDefaults) {
     loadFromSystemProperties(false)
   }
+  // tentatively enable offHeap for test
+  set("spark.memory.offHeap.enabled", "true")
+  set("spark.memory.offHeap.size", "1gb")
```

Comment:
Yes, it should be removed before merging this PR.
This code is tentative added to enable offheap mode.